packages/core-mobile/app/services/fcm/FCMService.ts

```diff
@@ -21,10 +22,17 @@ class FCMService {
       Logger.info('A new FCM message arrived!', remoteMessage)
       if (this.isValidRemoteMessage(remoteMessage)) {
         //TO DESIGN: show in-app notification instead of native
+        const result = NotificationsBalanceChangeSchema.safeParse(remoteMessage)
```

Comment:
the schema check should be inside the isValidRemoteMessage func

Reply:
actually, removed isValidRemoteMessage
[Update isValidRemoteMessage call](https://github.com/ava-labs/core-mobile/pull/1659/commits/68f50918ad6717267ea3d2741e2de5636b749147)